### PR TITLE
Add WobblyGuide

### PR DESCRIPTION
**Pull request by LightArrowsEXE**

### CODE REVIEW

File: DNS/WobblyGuide.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "name": "wobbly.encode.moe.",
+        "type": "A",
+        "value": "185.199.111.153",
+        "proxied": true
+    }
```
Comment (FichteFoll) [inline DNS/WobblyGuide.json:7]: Other files (like https://github.com/Irrational-Encoding-Wizardry/infra/blob/main/DNS/Guide.json) have 4 A entries (and I remember these instructions generally being the same for all gh pages sites).

Comment (LightArrowsEXE) [reply DNS/WobblyGuide.json:7]: I wasn't sure what to do about that or how to pick the values, so I figured I'd leave it to cid to explain to me.

Comment (FichteFoll) [reply DNS/WobblyGuide.json:7]: It's a way of load balancing since any (proper) DNS client will pick one of the 4 IPs randomly.

Comment (cid-chan) [reply DNS/WobblyGuide.json:7]: Should I commit, or are you going to be updating your PR?

**Pull request merged**